store/src/java/com/zimbra/cs/util/AccountUtil.java

```diff
@@ -1042,4 +1044,19 @@ public static void isAdminAccount(Account acct) throws ServiceException {
         if (!isAdminAccount)
             throw ServiceException.PERM_DENIED("not an admin account");
     }
+    
+    /**
+     * Utils method to translate system folder name
+     * @param name
+     * @param locale
+     * @return
+     */
+    public static String getTranslatedFolderName(String name, Locale locale) {
+        if ("Calendar".equals(name)) {
+            return L10nUtil.getMessage(L10nUtil.MsgKey.calendar, locale);
+        } else if ("Trash".equals(name)) {
+            return L10nUtil.getMessage(L10nUtil.MsgKey.calendar, locale);
```

Comment:
trash is also using calendar?

Reply:
> trash is also using calendar?

It should be trash as if operation is for 'Trash' folder then variable 'name' will contain value 'Trash' and this should be translated.